cdap-ui/app/hydrator/templates/partial/node-config-modal/node-config/plugin-edit-properties-template.html

```diff
@@ -14,44 +14,61 @@
       <small class="text-danger" ng-if="HydratorPlusPlusNodeConfigCtrl.state.nodeLabelError">{{HydratorPlusPlusNodeConfigCtrl.state.nodeLabelError}}</small>
     </label>
-    <input type="text" class="form-control"
+    <div class="col-sm-8">
+      <input type="text" class="form-control"
            ng-disabled="isDisabled"
            ng-model="HydratorPlusPlusNodeConfigCtrl.state.node.plugin.label">
+    </div>
   </div>
 
   <div ng-repeat="group in HydratorPlusPlusNodeConfigCtrl.state.groupsConfig.groups">
     <div class="widget-group-container">
-      <h4>{{::group.display}}</h4>
+      <div class="field-header subtitle">
+        <h4>{{::group.display}}</h4>
+      </div>
       <div ng-repeat="field in group.fields">
         <div ng-if="field.name !== HydratorPlusPlusNodeConfigCtrl.state.groupsConfig.outputSchema.schemaProperty">
 
-          <div class="form-group">
-            <label class="control-label">
-              <span>{{::field.label}}</span>
-
-              <span class="fa fa-info-circle" uib-tooltip="{{::field.description}}" tooltip-placement="right" tooltip-append-to-body="true">
+        <div class="form-group">
+          <div class="col-sm-offset-8 col-sm-4">
+              <span ng-if="field['plugin-function'] && (!field['plugin-function']['position'] || field['plugin-function']['position'] === 'top') && !isDisabled">
+                <plugin-functions fn-config="field['plugin-function']" node="HydratorPlusPlusNodeConfigCtrl.state.node">
+                </plugin-functions>
               </span>
-              <span class="fa fa-asterisk" ng-if="HydratorPlusPlusNodeConfigCtrl.state.node._backendProperties[field.name].required"></span>
-              <small class="text-danger" ng-if="!HydratorPlusPlusNodeConfigCtrl.state.node.warning && HydratorPlusPlusNodeConfigCtrl.state.node._backendProperties[field.name].required && !HydratorPlusPlusNodeConfigCtrl.state.node.plugin.properties[field.name]">{{::HydratorPlusPlusNodeConfigCtrl.requiredPropertyError}}</small>
+            </div>
+        </div>
+          <div class="form-group">
+            <label
+              class="col-sm-3 control-label"
+              ng-if="(['javascript-editor', 'python-editor'].indexOf(field['widget-type']) === -1)"
+            >
+              <span ng-class="{'required': HydratorPlusPlusNodeConfigCtrl.state.node._backendProperties[field.name].required}">{{::field.label}}</span>
             </label>
 
-            <span ng-if="field['plugin-function'] && (!field['plugin-function']['position'] || field['plugin-function']['position'] === 'top') && !isDisabled">
-              <plugin-functions fn-config="field['plugin-function']" node="HydratorPlusPlusNodeConfigCtrl.state.node">
-              </plugin-functions>
-            </span>
-
-            <div>
+            <div
+              ng-class="{
+                'col-sm-8 form-input-attr': (['javascript-editor', 'python-editor'].indexOf(field['widget-type']) === -1),
```

Comment:
Can still keep `class="form-input-attr"` and only do `ng-class` for `col-sm-8` and `col-sm-11`